parquet-column/src/main/java/org/apache/parquet/column/statistics/BinaryStatistics.java

```diff
@@ -19,12 +19,37 @@
 package org.apache.parquet.column.statistics;
 
 import org.apache.parquet.io.api.Binary;
+import org.apache.parquet.schema.PrimitiveType;
+import org.apache.parquet.schema.Types;
 
 public class BinaryStatistics extends Statistics<Binary> {
 
+  // A fake type object to be used to generate the proper comparator
+  private static final PrimitiveType DEFAULT_FAKE_TYPE = Types.optional(PrimitiveType.PrimitiveTypeName.BINARY).named("");
```

Comment:
There are still "fake" types that are unnamed. Please add names to all fake types like this one.

Reply:
Sorry, but I could not find any other. In every Statistics classes they are renamed to `DEFAULT_FAKE_TYPE`.

Reply:
I see the confusion. I'm not talking about the variable name. I'm talking about the type name: `.named("")`. That's what I'm concerned about leaking into error messages and other places.